build/Settings.cs

```diff
@@ -56,5 +57,73 @@ private static string config(string @default = null, [CallerMemberName] string k
         public static string CommitId => config(null, "Build.SourceVersion") ?? config("N /A", "APPVEYOR_REPO_COMMIT");
 
         public static string BuildArtifactsStorage => config(null);
+
+        public class SignInfo
+        {
+            public static string AzureSigningConnectionString => config(null, "AzureBlobSigningConnectionString");
+            public static readonly string AzureToSignContainerName = "azure-functions-cli";
+            public static readonly string AzureSignedContainerName = "azure-functions-cli-signed";
+            public static readonly string AzureSigningJobName = "signing-jobs";
+            public static readonly string ToSignZipName = $"{BuildNumber}-authenticode.zip";
+            public static readonly string ToSignThirdPartyName = $"{BuildNumber}-third.zip";
+            public static readonly string ToSignDir = "unsigned";
+            public static readonly string SignedDir = "signed";
+            public static readonly string Authenticode = "SignAuthenticode";
+            public static readonly string ThirdParty = "Sign3rdParty";
+            public static readonly string[] RuntimesToSign = new[] {"min.win-x86"};
+            public static readonly string[] filterExtenstionsSign = new[] { ".json", ".spec", ".cfg", ".pdb", ".config", ".nupkg", ".py" };
+
+            public static readonly string[] authentiCodeBinaries = new[] {
+                "DurableTask.AzureStorage.Internal.dll",
```

Comment:
Are we referring the Durable package directly. This could cause issues if we have breaking changes in durable extensions.